contracts/example/AEcosystem.sol

```diff
@@ -7,23 +7,23 @@ contract AEcosystem is APermissioned {
     // Constructor code
   }
 
-  function createProposal(address _permissionOwner) external allow(_permissionOwner, PERMISSION_CREATE) {
+  function createProposal(address _permissionOwner) external allow(_permissionOwner, PERMISSION_CREATE) notBlacklisted {
```

Comment:
I would have reversed the two to first check if the person is blacklist before applying the other modifier

Reply:
the two modifers*
```solidity
function createProposal(address _permissionOwner) external notBlacklisted allow(_permissionOwner, PERMISSION_CREATE)  {
```

Reply:
I will update